responsibleai/responsibleai/_internal/_forecasting_wrappers.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+# All wrappers in this file will be migrated to ml-wrappers.
+
+import inspect
+import sys
+
+import pandas as pd
+from ml_wrappers.model.base_wrapped_model import BaseWrappedModel
+
+from responsibleai.rai_insights.constants import ModelTask
+
+from .constants import _Forecasting
+
+_AZUREML = "azureml"
+_SKTIME = "sktime"
+
+
+def _get_model_package(model):
+    module = inspect.getmodule(model)
+    module_name = module.__name__.partition('.')[0]
+    model_package = sys.modules[module_name].__package__
+    return model_package
+
+
+def _wrap_model(model, examples, time_feature, time_series_id_features):

```

Comment:
this definitely looks like something that should go in the ml-wrappers repository eventually

Reply:
Yes! I'll open a PR there. Once it's part of a ml-wrappers release I'll just point there and delete this file. I made sure to use all leading underscores here so that nobody starts relying on it.